src/json/to_json.cr

```diff
@@ -188,7 +188,24 @@ module JSON::ArrayConverter(Converter)
   end
 end
 
-module JSON::HashConverter(Converter)
+# Converter to be used with `JSON.mapping`
```

Comment:
These descriptions mentioning `Time` are IMO very confusing. One could think that's the only use case, whereas IIUC that's supposed to be the example.

Reply:
True. When writing it I had in mind only the `Time` converters.